src/routes/index.ts

```diff
@@ -4,18 +4,28 @@ import {
   deleteUser,
   getAllUsers,
   getUserById,
-  updateUser,
 } from '../controllers/user.controller';
 import { validateSchema, UserSchema } from '../validations/user.validation'
+import { isAuthorized } from '../middlewares/user.authenticate';
+import { UserRole } from '../models/user.model';
+import { roleController } from '../controllers/role.controller';
+
+
+
+
 
 const router = express.Router();
 
 export default (): express.Router => {
   router.post('/create', validateSchema(UserSchema.signUp), registerUser);
   router.get('/users', getAllUsers);
   router.get('/:id', getUserById);
```

Comment:
None of this should be here. Can you please check how other index files are used?
```suggestion
```